WTWSMS/common/minor_titles/WtWSMS_imperial_legacy_minor_titles.txt

```diff
@@ -466,11 +492,12 @@ title_il_comes_sacrae_vestis = {
 	
 	allowed_to_grant = {
 		is_feudal = yes
-		AND = {
+		culture = roman
+		OR = {
 			has_landed_title = e_wre
-			culture = roman
+			has_landed_title = e_roman_empire
 		}
-	}		
+	}
 
 	allowed_to_hold = {
 		is_adult = yes
```

Comment:
I didn't change that. What I added was that the Restored Roman Empire could use ERE AND WRE titles. Everything else stayed the same (must be Roman culture, must hold RE, WRE, or ERE, etc.).